src/OpenLoco/Viewport.hpp

```diff
@@ -112,10 +112,8 @@ namespace OpenLoco::Ui
             return out;
         }
 
-        constexpr int getRotation() const
-        {
-            return Interop::addr<0x00e3f0b8, int32_t>();
```

Comment:
This change means the `#include "Interop/Interop.hpp"` can be dropped from this file. (Great!)